seaworthy/fixtures.py

```diff
@@ -0,0 +1,40 @@
+from seaworthy.containers.redis import RedisContainer
+from seaworthy.containers.rabbitmq import RabbitMQContainer
+from seaworthy.definitions import ContainerDefinition
+from seaworthy.pytest.fixtures import resource_fixture
+
+
+class JunebugContainer(ContainerDefinition):
+    IMAGE = 'praekeltfoundation/junebug'
+    WAIT_PATTERNS = (
+        'Junebug is listening on',
+    )
+
+    def __init__(self, name, create_kwargs):
+        super().__init__(name, self.IMAGE, self.WAIT_PATTERNS,
+                         create_kwargs=create_kwargs)
+
+redis_fixture = resource_fixture(RedisContainer(), 'redis')
```

Comment:
You can do `RedisContainer().pytest_fixture('redis')` now. Where did you see an example of `resource_fixture`?

Reply:
I found this looking at the tests I think. I used the docs and your demo app too.

Reply:
Ah I see we've got it in the intro 🤦‍♂️.

`pytest_fixture` does exactly the same thing but is slightly more natural so we've been trying to update the examples to use that.